ascii/ascii.go

```diff
@@ -59,7 +71,7 @@ func NewASCII(dat string) (*ASCII, error) {
 			if fg == -1 {
 				// Reset color codes.
 				fg, bg = -1, -1
-				continue
+				break
```

Comment:
add test case in ascii_test.go? I don't understand what this particular change fixes

Reply:
without the break it would eat a character after a \x03 to reset colors because it wasn't a number